### PR TITLE
Dune index: cleanup models

### PR DESCRIPTION
**Pull request by 0xRobin**
## Thank you for contributing to Spellbook 🪄
Please open the PR in **draft** and mark as ready when you want to request a review. 

### Description:

Dune index: cleanup models
- Split out transactions by chain
- Split out fees by chain
- Split out transfers by chain

---
quick links for more information:
- [README.md](https://github.com/duneanalytics/spellbook/blob/main/README.md)
- [spellbook docs](https://github.com/duneanalytics/spellbook/tree/main/docs)
- [CONTRIBUTING.md](https://github.com/duneanalytics/spellbook/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: dbt_subprojects/daily_spellbook/models/_metrics/dune_index/_schema.yml [hunk 1]
```diff
@@ -13,10 +13,3 @@ models:
           combination_of_columns:
             - blockchain
             - block_date
-  - name: metrics_dune_index_stats
-    meta:
-      sector: metrics
-      contributors: jeff-dude
-    config:
-      tags: ['metrics', 'dune', 'index', 'daily']
-    description: "View containing various time aggregations of the dune index"
```
Comment (jeff-dude) [inline dbt_subprojects/daily_spellbook/models/_metrics/dune_index/_schema.yml:22]: should likely restore this

File: dbt_subprojects/daily_spellbook/models/_metrics/fees/_schema.yml [hunk 2]
```diff
@@ -8,27 +8,3 @@ models:
...
-      sector: metrics
-      contributors: jeff-dude
-    config:
-      tags: ['metrics', 'fees', 'gas', 'index', 'daily']
-    description: "Each day, per chain, compare the adoption of activity based on total gas fees relative to the baseline expectation"
-    data_tests:
-      - dbt_utils.unique_combination_of_columns:
-          combination_of_columns:
-            - blockchain
-            - block_date
```
Comment (jeff-dude) [inline dbt_subprojects/daily_spellbook/models/_metrics/fees/_schema.yml:34]: should restore this

File: dbt_subprojects/daily_spellbook/models/_metrics/transactions/_schema.yml [hunk 3]
```diff
@@ -8,27 +8,3 @@ models:
...
-      sector: metrics
-      contributors: jeff-dude
-    config:
-      tags: ['metrics', 'transactions', 'index', 'daily']
-    description: "Each day, per chain, compare the adoption of total transactions relative to the baseline expectation"
-    data_tests:
-      - dbt_utils.unique_combination_of_columns:
-          combination_of_columns:
-            - blockchain
-            - block_date
```
Comment (jeff-dude) [inline dbt_subprojects/daily_spellbook/models/_metrics/transactions/_schema.yml:34]: should restore this

File: dbt_subprojects/daily_spellbook/models/_metrics/transfers/_schema.yml [hunk 4]
```diff
@@ -47,35 +44,3 @@ models:
...
-      sector: metrics
-      contributors: jeff-dude
-    config:
-      tags: ['metrics', 'transfers', 'index', 'daily']
-    description: "Each day, per chain, compare the adoption of total transfers relative to the baseline expectation"
-    data_tests:
-      - dbt_utils.unique_combination_of_columns:
-          combination_of_columns:
-            - blockchain
-            - block_date
```
Comment (jeff-dude) [inline dbt_subprojects/daily_spellbook/models/_metrics/transfers/_schema.yml:81]: should restore this

**Review by jeff-dude — APPROVED** [review]
approving with two outstanding items / questions:
- above minor comments
- have you tested the CI query outputs with the queries used on app, using your private query to quickly validate data there? maybe we can fork that query and apply CI tables, even if only a few days of data?

**Comment by 0xRobin** [comment]
Results between CI <> prod look the same :heavy_check_mark: 
image
image
image

**Pull request merged**